### ISSUE TITLE
Reader Mode's display settings should be focused by accessibility after showing it, not allow elements outside to be pressed

### ISSUE THREAD

**Issue by mozillamobilebot** [mozilla-mobile/firefox-ios#4980]
From https://bugzilla.mozilla.org/show_bug.cgi?id=1160640

User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_3) AppleWebKit/600.5.17 (KHTML, like Gecko) Version/8.0.5 Safari/600.5.17

Steps to reproduce:

Open Reader View
Launch VoiceOver
Activate the Display settings button for reader view

Actual results:

VoiceOver cursor moved to reader view contents

Expected results:

VoiceOver cursor would move to display settings popover

**Comment by SimonBasca** [comment]
This is no longer reproducible.
